stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -444,22 +462,28 @@ ::util::Status GetAllEntries(
       RETURN_IF_TDI_ERROR(table->dataAllocate(&data[i]));
       pairs.push_back(std::make_pair(keys[i].get(), data[i].get()));
     }
-    uint32 actual = 0;
+    actual = 0;
```

Comment:
Instead of declaring `actual` at the top of the function and initializing it here, it is better C++ style to specify:
```
uint32 actual = 0;
```
This limits the scope of the variable to the time it is actually used.